dataplug/app/com/hubofallthings/dataplug/actors/DataPlugManagerActor.scala

```diff
@@ -176,7 +177,10 @@ class DataPlugManagerActor @Inject() (
 
           eventualSyncMessage.recover {
             case e =>
-              logger.error(s"Error while trying to start syncing dat for $phata, variant $variant: ${e.getMessage}", e)
+              logger.error(s"Error while trying to start syncing data for $phata, variant $variant: ${e.getMessage}", e)
+              val sw = new StringWriter
+              e.printStackTrace(new PrintWriter(sw))
+              logger.error(sw.toString)
```

Comment:
the two lines above add nothing to the first logging statement as it will print the stacktrace... also no need to repeat ${e.getMessage} it will be in the output.
just do `logger.error(s"Error while trying to start syncing data for $phata, variant $variant", th)`
`th` because it is a `Throwable` not just an exception (catching everything is not considered a good practice)

Reply:
Sorry I don't understand, are you saying the first logger will print the stack trace either way? If yes it doesn't 😅 

PS: I agree about the `e.getMessage` that is not needed.

Reply:
and why is it working in devdashboard then ?

Reply:
maybe because the log pattern is wrong (as far as I am concerned) in dataplugs:
      <pattern>%logger{15} - %message%n%xException{10}</pattern>

Reply:
for information https://www.baeldung.com/slf4j-log-exceptions

Reply:
actually the existing pattern in dataplugs works (tested using the devdash integration tests):
```
[info] FileUploadMigrationAppSpec:
s.b.DatabaseConfig - Use `profile` instead of `driver`. The latter is deprecated since Slick 3.2 and will be removed.
i.d.d.d.MigrationRunner - data migration failed
com.amazonaws.services.s3.model.AmazonS3Exception: The specified bucket does not exist (Service: Amazon S3; Status Code: 404; Error Code: NoSuchBucket; Request ID: null; S3 Extended Request ID: null; Proxy: null)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleErrorResponse(AmazonHttpClient.java:1819)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleServiceErrorResponse(AmazonHttpClient.java:1403)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1372)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1145)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:802)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:770)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:744)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:704)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:686)
	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:550)
```